docs: troubleshooting on browser extension blocker (#12737)
vitejs/vite: docs/guide/troubleshooting.md

> TypeError: Cannot create property 'foo' on boolean 'false'

If these code are used inside dependencies, you could use [`patch-package`](https://github.com/ds300/patch-package) (or [`yarn patch`](https://yarnpkg.com/cli/patch) or [`pnpm patch`](https://pnpm.io/cli/patch)) for an escape hatch.

> ### Browser extension blocker
> 
> Some browser extensions(e.g ad-blocker) may prevent vite client from sending requests to vite dev server. This might make the vite client page become white screen without any errors output. Try to disable these extensions before you visit the page.

Reviewer: ```suggestion
### Browser extensions

Some browser extensions (like ad-blockers) may prevent the Vite client from sending requests to the Vite dev server. You may see a white screen without logged errors in this case. Try disabling extensions if you have this issue.
```
